sphinx/config.py

```diff
@@ -217,6 +217,12 @@ def convert_overrides(self, name, value):
             defvalue = self.values[name][0]
             if self.values[name][2] == Any:
                 return value
+            elif type(self.values[name][0]) is bool or self.values[name][2] == [bool]:
```

Comment:
perhaps use `defvalue` here in place of `self.values[name][0]`

Reply:
fixed. thanks!